Automattic/jetpack
# packages: Add changeloger tool package

#### Changes proposed in this Pull Request:

Add a "changelogger" package, which is a tool to manage changelogs via having each PR drop a change file into a "changelog" directory.

Note I've liberally ignored the WordPress file and variable naming rules here. The rest of the PHP universe has settled on PSR-4 and camelCase for method names (and usually variable names too) and since this package has nothing to do with WordPress I chose to follow the wider ruleset.

#### Jetpack product discussion


p9dueE-2qL-p2
1199658893173824-as-1199910496214526

#### Does this pull request change what data or activity we track or use?


No.

#### Testing instructions:






* Does CI pass? I wrote lots of tests.
* Try it out, see if it works.
* Check that the readme makes sense and has no typos.

#### Proposed changelog entry for your changes:


* None needed.
